emoji_message.py

```diff
@@ -0,0 +1,61 @@
+"""Constructs emoji report message."""
+
+
+class EmojiMessage:
+    """Constructs emoji report message."""
+
+    def __init__(self, channel, icon_emoji, event_type):
+        """Initialize EmojiMessage.
+
+        Created with the channel to send the message to, the event type that triggered the callback and the emoji to
+        report on.
+
+        Arguments:
+            channel {string}    -- The channel to send the emoji message to
```

Comment:
Stick with str, not string for these I think. Later we can add actual typing. :smiling_imp:

Reply:
auto gen /shrug